core/templates/dev/head/services/rte-helper.service.ts

```diff
@@ -129,10 +135,15 @@ angular.module('oppia').factory('RteHelperService', [
                 var customizationArgsDict = {};
                 for (var i = 0; i < $scope.tmpCustomizationArgs.length; i++) {
                   var caName = $scope.tmpCustomizationArgs[i].name;
-                  customizationArgsDict[caName] = (
-                    $scope.tmpCustomizationArgs[i].value);
+                  if (caName === 'video_id') {
```

Comment:
When would this not be 'video_id'? Since I don't see any changes that would filter just video ID and video URL... Can you explain what the mechanism here is?

Reply:
@aks681, `caName` depends on the `tmpCustomizationArgs[i].value` ,In the case of video upload, It contains
`video_id`, `start`, `end`, `autoplay`.(pls look at the customization args for video in `assets\rich_text_components_definitions.ts`)
So we filter based on caName , and when the caName is `video_id` we extract the video_id based on what user has entered(the url or the video_id itself).

Reply:
> When would this not be 'video_id'

This would change based on the RTE component we are editing .